app/models/concerns/foreman_openscap/policy_extensions.rb

```diff
@@ -143,6 +155,10 @@ def to_enc
       }.merge(period_enc)
     end
 
+    def find_assets(method)
```

Comment:
To be honest, I'm a little uncomfortable with this method, as it can send anything to ```Scaptimony::Asset```.
I think that something like: ```policy.assets.incomply_with(policy)``` || ```policy.assets.comply_with(policy)```  would be sufficient for this use case. @ares - would love to hear your comments on this.